oxytcmri/tests/unit/domain/mocks.py

```
@@ -7,7 +7,8 @@
     DTIMetric,
     Atlas,
     MRIExam,
-    VoxelData, AtlasSegmentation, DTIMap, T, MRIExamId, MRIData, Mask,
+    VoxelData, AtlasSegmentation, DTIMap, T, MRIExamId, MRIData, Mask, RegionOfInterest, DTIAbnormalValues,
+    AbnormalVoxelData, AbnormalValueType,
```

Ruff (0.11.9)

`oxytcmri.domain.entities.mri.AbnormalVoxelData` imported but unused

Remove unused import: `oxytcmri.domain.entities.mri.AbnormalVoxelData`

(F401)